packages/rewriter/script/worker-shim.js

```diff
@@ -3,13 +3,8 @@ if (process.env.CYPRESS_INTERNAL_ENV === 'production') {
   throw new Error(`${__filename} should only run outside of prod`)
 }
 
-if (require.name !== 'customRequire') {
-  // Purposefully make this a dynamic require so that it doesn't have the potential to get picked up by snapshotting mechanism
-  const hook = './hook'
+const { hookRequire } = require('@packages/server/hook-require')
 
-  const { hookRequire } = require(`@packages/server/${hook}-require`)
-
-  hookRequire(true)
-}
+hookRequire({ forceTypeScript: true })
```

Comment:
how do we know to force typescript?

Reply:
This file never runs in prod (see line 2). It only runs in development where we still need to force typescript compilation on the fly.